### ISSUE TITLE
Cannot switch to private net

### ISSUE THREAD

**Issue by habtt-ibl** [CityOfZion/neo-scan#378]
I built a private net on my servers. How can I configure the neo-scan to **connect to my own private-net**?

**Comment by QuangVUHuflit** [comment]
Yes, I am trying to ask this question. Can anyone share the config for phoenix to work with the private net? Thanks

**Comment by adrienmo** [comment]
can you use neo-local project for this?

https://github.com/CityOfZion/neo-local

**Comment by simonemanneschi** [comment]
Just use env variable NEO_SEEDS and NEO_NOTIFICATIONS_SERVER pointing to your private instance
